config/SuperBuild/TPLVersions.cmake

```diff
@@ -172,20 +173,19 @@ set(METIS_VERSION_MAJOR 5)
 set(METIS_VERSION_MINOR 1)
 set(METIS_VERSION_PATCH 0)
 set(METIS_VERSION ${METIS_VERSION_MAJOR}.${METIS_VERSION_MINOR}.${METIS_VERSION_PATCH})
-set(METIS_URL_STRING     "http://software.lanl.gov/ascem/tpls")
+set(METIS_URL_STRING     "https://raw.githubusercontent.com/amanzi/amanzi-tpls/master/src")
 set(METIS_ARCHIVE_FILE   metis-${METIS_VERSION}.tar.gz)
 set(METIS_MD5_SUM        5465e67079419a69e0116de24fce58fe)
 
 #
 # TPL: UnitTest
 #
 set(UnitTest_VERSION_MAJOR 1)
-set(UnitTest_VERSION_MINOR 6)
-set(UnitTest_VERSION_PATCH 0)
-set(UnitTest_VERSION ${UnitTest_VERSION_MAJOR}.${UnitTest_VERSION_MINOR}.${UnitTest_VERSION_PATCH})
-set(UnitTest_URL_STRING     "http://software.lanl.gov/ascem/tpls")
+set(UnitTest_VERSION_MINOR 5)
```

Comment:
This is the version that's used by Amanzi, and the newer one didn't seem to be available for download from that URL.